enos/modules/upgrade_client/scripts/verify_allocs.sh

```diff
@@ -11,6 +11,7 @@ error_exit() {
 
 MAX_WAIT_TIME=60  # Maximum wait time in seconds
 POLL_INTERVAL=2   # Interval between status checks
+CLIENT_IP=54.161.192.157
```

Comment:
Do we want to remove this IP address (assuming it was for debugging)?

Reply:
OMG! YES!!! Thank you!!!